src/components/alert/alert/readme.md

```diff
@@ -73,6 +73,11 @@ The description text is wrapped into an `<p>` element to guarantee the semantic
 
 Avoid slotting block elements (e.g. `<div>`) as this violates semantic rules and can have negative effects on screen readers.
 
+## Animation
+
+By default, the `sbb-alert` does not have an open animation. Set `[animation]='open'` to enable a fade-in animation.
```

Comment:
outdated docs

Reply:
I'm not sure whether to keep the animation section at all.
I went with:

"Set the `animation` property to manage which animations are enabled.
As a base rule, opening animations should be active if a notification arrives after the initial page load."